skimage/io/_plugins/tifffile_plugin.py

```diff
@@ -1,7 +1,7 @@
-from ...external.tifffile import TiffFile, imsave
+from ...external.tifffile import TiffFile, imsave, parse_kwargs
```

Comment:
According to our tests, this `parse_kwargs` function doesn't exist...?

Reply:
It isn't exported from `tiffille.__init__` https://github.com/scikit-image/scikit-image/blob/master/skimage/external/tifffile/__init__.py

You need to import it from `external.tifffile.tifffile`